db/migrate/20210707115300_add_default_timestamps_to_keywords.rb

```diff
@@ -0,0 +1,10 @@
+class AddDefaultTimestampsToKeywords < ActiveRecord::Migration[6.1]
+  def up
+    execute("ALTER TABLE keywords ALTER COLUMN created_at SET DEFAULT CURRENT_TIMESTAMP;")
+    execute("ALTER TABLE keywords ALTER COLUMN updated_at SET DEFAULT CURRENT_TIMESTAMP;")
+  end
+  def down
```

Comment:
A blank line can be added between these 2 methods.

Reply:
Woups!